index.html

```diff
@@ -14,6 +14,68 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header-container">
+        <img src="./img/first-background.jpg" alt="">
+        <nav class="nav-bar">
+            <img class="karma-logo" src="./img/karma-logo.svg" alt="karma-logo">
+            
+                    <ul>
+                        <li> <a href="#"> Meet Karma </a></li>
+                        <li><a href="#"> How it works </a></li>
+                        <li><a href="#"> Store </a></li>
+                        <li><a href="#"> Blog </a></li>
+                        <li><a href="#"> Help </a></li>
+                        <li><a href="#"> Login </a></li>
+                    </ul>
+                </nav>
+
+        <div class="introducing-karma">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go.</p>
+        <div class="learn-more"> <a href="#">Learn more</a> </div>
+        </div>
+    </header>
+
+    <main>
+        <div class="main-container">
```

Comment:
```suggestion
    <main class="main-container">
```
We don't need the extra `div` inside `main`, so we can add the class to the `main` instead and delete the `div`